README.md

```diff
@@ -99,3 +100,6 @@ npm test
 ## License
 
 This project is distributed under the MIT license.
+
+[appveyor-badge]: https://ci.appveyor.com/api/projects/status/7owf61lo8uujbjok/branch/master?svg=true
```

Comment:
I don't think this token is correct @stefanpenner. I'll be happy to change once `appveyor` is enabled for the project.